Acrolinx.Sidebar.SDK/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "version": "1.0.0",
+  "name": "cpp-sidebar-sdk",
+  "private": true,
+  "devDependencies": {
+    "@acrolinx/sidebar-startpage": "3.1.1"
```

Comment:
version should late latest minor version

Reply:
use semantic versioning